lib/extensions/ar_migration.rb

```diff
@@ -1,6 +1,7 @@
 module ArPglogicalMigrationHelper
   def self.discover_schema_migrations_ran_class
-    return unless ActiveRecord::Base.connection.table_exists?("schema_migrations_ran")
+    raise "Remove 'silence_warnings' block now that Rails version is >= 5.1" if Rails.version >= "5.1"
```

Comment:
can this be warn so my branch doesn't fail all over?

Reply:
Can you revert this in your 5-1 branch instead?

Reply:
I'm a jerk.  The error says `silence_warnings` but we changed it to `silence`...

🏃